lib/swaps/Swaps.ts

```diff
@@ -433,6 +433,29 @@ class Swaps extends EventEmitter {
 
     const { makerCurrency, makerAmount, takerCurrency, takerAmount } = Swaps.calculateMakerTakerAmounts(quantity, price, isBuy, requestBody.pairId);
 
+    const makerSwapClient = this.swapClientManager.get(makerCurrency)!;
+    if (!makerSwapClient) {
+      await this.sendErrorToPeer({
+        peer,
+        rHash,
+        failureReason: SwapFailureReason.SwapClientNotSetup,
+        errorMessage: 'Unsupported maker currency',
+        reqId: requestPacket.header.id,
+      });
+      return false;
+    }
+
+    if (isRaidenClient(makerSwapClient)) {
```

Comment:
I assume this is temporary logic that's going to be removed in #1082?

Reply:
Yes.